Volume/Supporting/Notifications.swift

```diff
@@ -98,7 +98,7 @@ extension Notifications {
 }
 
 extension Notifications: MessagingDelegate {
-    func messaging(_ messaging: Messaging, didReceiveRegistrationToken fcmToken: String?) {
```

Comment:
Not sure if this is a UIKIT only thing but we usually have a linebreak here after defining the extension and also at the line before the extension ends

Reply:
Right. This was written before me but thanks for pointing it out.